tmt/utils.py

```diff
@@ -56,6 +56,9 @@
 # Default select.select(timeout) in seconds
 DEFAULT_SELECT_TIMEOUT = 5
 
+# Shell options to be set for all run shell scripts
+SHELL_OPTIONS = 'set -eo pipefail'
```

Comment:
Why is there a duplicity with `run.sh`?

Reply:
Does it mirror the defaults `run.sh`? Is there a way to pass this to a script, or is that not an expected usage? (Is it supposed to be configurable?)

Reply:
Thanks for explanation!